### DISCUSSION TITLE
Black screen on 2.0.3 version of code

### DISCUSSION THREAD

**Discussion by Kliuchko** [sceneview/sceneview#514] [Q&A] [answered]
Does anyone know what's going on with version 2.0.3? (io.github.sceneview:arsceneview:2.0.3)
In 2.0.2 everything I use works fine, but in 2.0.3 I launch the application and a black screen appears instead of the camera. I can see the planeRenderer points but the screen is black and I can also render the view. Can anyone help me with this?

**Comment by SyedMoizoffical** [comment]
check configureSession comment this (config.textureUpdateMode=Config.TextureUpdateMode.EXPOSE_HARDWARE_BUFFER)
if issue not resolve comment configureSession  ,config One by one

**Comment by Kliuchko** [comment]
I don’t use these methods because I don’t configure the session in a special way; for my case I need a basic configuration of ARCore

**Comment by SyedMoizoffical** [comment]
Check Camera permission ,it Allow

**Comment by Kliuchko** [comment]
Yes, everything is allow

**Comment by Kliuchko** [comment]
Any thoughts?

**Comment by SyedMoizoffical** [comment]
back to 2.0.2

**Comment by ridcore** [comment, marked as answer]
Black screen indicates some type of error, check your logcat and tell us if it gives you any errors

**Comment by Kliuchko** [comment]
@devR2id Didn't find any errors
